build/changelog.py

```diff
@@ -100,12 +121,10 @@ def fixes_issue_id(commit_message):
 def get_subject(commit_message):
     return commit_message.splitlines()[0]
 
-
-def get_changelog_message(commit_message, mention, repo_url):
-    issue_id = fixes_issue_id(commit_message)
+def get_changelog_message(commit_message, issue_id, mention, reporter, repo_url):
     if issue_id:
         subject = get_subject(commit_message)
-        return "Fixes", "{subject} ([#{issue_id}]({repo_url}/issues/{issue_id})) {mention}".format(subject=subject, issue_id=issue_id, repo_url=repo_url, mention=mention)
+        return "Fixes", "{subject} ([#{issue_id}]({repo_url}/issues/{issue_id})) {mention}{reporter} ".format(subject=subject, issue_id=issue_id, repo_url=repo_url, reporter=reporter, mention=mention)
```

Comment:
Are we relying on some old Python version here? If not, f style strings would make this more readable IMO.

https://realpython.com/python-f-strings/

Reply:
We can't use f-strings without updating the docker image (`python:2.7`) that's used to run this script. Let's avoid that since I don't think it's worth the trouble for now.